superset-frontend/src/components/EditableTitle.tsx

```diff
@@ -23,28 +23,33 @@ import TooltipWrapper from './TooltipWrapper';
 
 interface EditableTitleProps {
   canEdit?: boolean;
+  editing?: boolean;
   emptyText?: string;
   extraClasses?: Array<string> | string;
   multiLine?: boolean;
   noPermitTooltip?: string;
   onSaveTitle: (arg0: string) => {};
   showTooltip?: boolean;
   style?: object;
-  title: string;
+  title?: string;
+  defaultTitle?: string;
+  placeholder?: string;
 }
 
 export default function EditableTitle({
   canEdit = false,
-  emptyText,
+  editing = false,
   extraClasses,
   multiLine = false,
   noPermitTooltip,
   onSaveTitle,
   showTooltip = true,
   style,
-  title,
+  title = '',
+  defaultTitle = '',
+  placeholder = '',
```

Comment:
Do we need both defaultTitle AND placeholder, or could these share a common purpose?

Reply:
they serve the same purpose right from the UI?

Reply:
@junlincc @rusackas the component is used by several others. In order to safely apply these changes to the tabs titles only, a separation between title and defaultTitle needs to be made. If we want to change them all in the future, this can be consolidated and one of the two removed. That is not the case now though.